src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -505,6 +505,33 @@
     InventoryArchetype getArchetype();
 
     /**
+     * Intersects the slots of both inventories
```

Comment:
This needs more documentation as it isn't clear what is going on whatsoever here.